SignalShareExtension/ShareViewController.swift

```diff
@@ -214,6 +215,8 @@ public class ShareViewController: UINavigationController, SAELoadViewDelegate {
             //                SignalsNavigationController *navigationController =
             //                    [[SignalsNavigationController alloc] initWithRootViewController:homeView];
             //                self.window.rootViewController = navigationController;
```

Comment:
Can we remove these commented-out lines?

Reply:
done!